### DISCUSSION TITLE
中文章节号从汉字变成了数字

### DISCUSSION THREAD

**Discussion by zijun-cs** [sjtug/SJTUThesis#1060] [提问] [answered]
**问题描述**:
中文章节号从汉字变成了数字，如 第一章 变成了 第1章

请问这是论文格式规范的新要求还是bug？

**Comment by LogCreative** [comment, marked as answer]
新的[参考模板](https://www.gs.sjtu.edu.cn/post/detail/Z3M2MjU=)使用阿拉伯数字进行章节编号。

如果想改为中文数字编号使用：

```latex
\ctexset{
  chapter/number=\chinese{chapter},
  chapter/name={第,章}
}
```

**Reply by lynnaero130** [reply]
但是这样的话，“第”和“一”之间会有个空格\捂脸

**Reply by LogCreative** [reply]
确实。已经更新上面的示例代码，现在的代码片段应该不会有空格问题。章节编号的变更参见 https://github.com/sjtug/SJTUTeX/commit/729f3515a9876ba7684cf0bf7539eb1722aad9fd 。

**Reply by lynnaero130** [reply]
可以了！感谢！！
